libs/api-bot/api-bot.cabal

```diff
@@ -0,0 +1,89 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.32.0.
```

Comment:
This line will make stack not re-generate cabal files if it was compiled with older hpack. We all have to agree on the version of hpack that stack was compiled with. The latest one is 0.33. Do we want that instead?

Reply:
that would be a good reason to get rid of hpack, because whatever we do, it'll always be pain.  :(